crates/api_models/src/admin.rs

```diff
@@ -1971,6 +1971,12 @@ pub struct ProfileCreate {
     /// If set to `true` is_network_tokenization_enabled will be checked.
     #[serde(default)]
     pub is_network_tokenization_enabled: bool,
+
+    /// Indicates if is_auto_retries_enabled is enabled or not.
+    pub is_auto_retries_enabled: Option<bool>,
+
+    /// Maximum number of auto retries allowed for a payment
```

Comment:
can have max value in the documentation

Reply:
Sure, will make this changes in my next PR